SPECS/libsoup/libsoup.spec

```diff
@@ -44,6 +44,16 @@ Requires:       libxml2
 Patch:          CVE-2024-52530.patch
 Patch:          CVE-2024-52531.patch
 Patch:          CVE-2024-52532.patch
+# CVE-2025-32913 will be fixed in 3.6.2 by https://gitlab.gnome.org/GNOME/libsoup/-/commit/f4a761fb66512fff59798765e8ac5b9e57dceef0
+Patch:          CVE-2025-32913.patch
```

Comment:
Can you please add patch numbers to each of these patches, to ensure they are applied a consistent/correct order?

Reply:
Done